.github/workflows/ci.yml

```diff
@@ -37,6 +37,12 @@ jobs:
           mkdir pkg/ui/{aim,mlflow}/embed/build
           touch pkg/ui/{aim,mlflow}/embed/build/index.html
 
+      - name: Download mockery
+        run: go install github.com/vektra/mockery/v2@v2.20.0
+
+      - name: Generate mocks
+        run: make mocks-generate; sleep 5;
```

Comment:
Why is there a `sleep 5` in there?

Reply:
it could be a small lag, so mocks appear with a small delay. 5 seconds enough for all files to appear.

Reply:
so just to make sure.

Reply:
do you mean that when `make mocks-generate` is done, it's actually not fully done? 🤔

Reply:
it seems like it works well without sleep but I had such a problem on my local. ok, then lets see.